slowish/models.py

```diff
@@ -75,3 +75,21 @@ class Meta:
 
     def __unicode__(self):
         return "{0} (in account {1})".format(self.name, self.account.id)
+
+
+class SlowishFile(models.Model):
+    """A file, (within a particular container)."""
+
+    container = models.ForeignKey(SlowishContainer, on_delete=models.CASCADE)
+
+    path = models.CharField(
+        help_text="Complete path of this file (within the container).",
+        max_length=255)
```

Comment:
255 seems like a reasonable length to me, but I wanted to point out that the [Swift documentation](http://docs.openstack.org/developer/swift/api/object_api_v1_overview.html#object-storage-api-overview) mentions that the default max length of object names (not including the container) is 1024 bytes. The [Rackspace docs](https://developer.rackspace.com/docs/cloud-files/v1/developer-guide/#objects) state that the max object length is 1024 bytes after URL encoding. Perhaps a safer value would be `max_length=1024`?